app/router.js

```diff
@@ -7,15 +7,10 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-    this.route('preprints', { path: '/preprints/:file_id' }, function() {
-        this.route('view', { path: '/' });
-        this.route('edit', { path: '/edit' });
```

Comment:
> No edit route?

Basically, there's a lack of design clarity on what editing a preprint would entail, and no one appears tasked to work on it. 

Based on our timeline, I removed the route so that it wouldn't be inadvertently accessible in the initial public release of the product.